<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background: rgb(67, 83, 106);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">background: rgba(51, 64, 88, 0%);</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="startPage">
       <property name="styleSheet">
        <string notr="true">background: rgb(67, 83, 106)</string>
       </property>
       <widget class="QHoverSensitiveButton" name="fetchButton">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>80</y>
          <width>271</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>30</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background: rgba(51, 64, 88, 0%);
border-style: solid;

</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/fetch.png</normaloff>:/images/fetch.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>180</width>
          <height>150</height>
         </size>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="settingButton">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>170</y>
          <width>131</width>
          <height>131</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background: rgba(51, 64, 88, 0%);
border-style: solid;

</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>20</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>.Helvetica Neue DeskInterface</family>
          <pointsize>20</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;.Helvetica Neue DeskInterface&quot;;
color: white;</string>
        </property>
        <property name="text">
         <string>Hover + Hold</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>50</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>.Helvetica Neue DeskInterface</family>
          <pointsize>20</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;.Helvetica Neue DeskInterface&quot;;
color: white;</string>
        </property>
        <property name="text">
         <string>to Select</string>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="tutorialButton">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>170</y>
          <width>131</width>
          <height>131</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background: rgba(51, 64, 88, 0%);
border-style: solid;

</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/tutorials.png</normaloff>:/images/tutorials.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>120</width>
          <height>130</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="movementPage">
       <widget class="QHoverSensitiveButton" name="releaseButton">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>230</y>
          <width>162</width>
          <height>102</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Helvetica</family>
          <pointsize>29</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(90, 225, 143, 0%);
border-style: solid;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/release.png</normaloff>:/images/release.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="backButton_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 0, 0, 0%);
border: solid;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/back.png</normaloff>:/images/back.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="forwardButton">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>200</y>
          <width>161</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(38, 48, 71, 0%);
font: 29pt &quot;Helvetica&quot;;
border: solid;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/forward.png</normaloff>:/images/forward.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="backwardButton">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>270</y>
          <width>161</width>
          <height>91</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(38, 48, 71, 0%);
font: 29pt &quot;Helvetica&quot;;
border: solid;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/backward.png</normaloff>:/images/backward.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="upButton">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>70</y>
          <width>111</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(38, 48, 71, 0%);
font: 20pt &quot;Helvetica&quot;;
border: solid;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/up.png</normaloff>:/images/up.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>80</width>
          <height>75</height>
         </size>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="leftButton">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>80</y>
          <width>71</width>
          <height>101</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(38, 48, 71, 0%);
font: 20pt &quot;Helvetica&quot;;
border: solid;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/left.png</normaloff>:/images/left.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>120</height>
         </size>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="rightButton">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>80</y>
          <width>71</width>
          <height>101</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(38, 48, 71, 0%);
font: 20pt &quot;Helvetica&quot;;
border: solid;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/right.png</normaloff>:/images/right.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>120</height>
         </size>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="downButton">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>140</y>
          <width>111</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(38, 48, 71, 0%);
font: 20pt &quot;Helvetica&quot;;
border: solid;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/down.png</normaloff>:/images/down.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>80</width>
          <height>75</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="ready_label">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>10</y>
          <width>201</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">qproperty-alignment: AlignCenter;
font: 20pt &quot;.Helvetica Neue DeskInterface&quot;;
color: white;
</string>
        </property>
        <property name="text">
         <string>Ready for commands!</string>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="clawLeft">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>230</y>
          <width>91</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-style: solid;
background-color: rgb(183, 183, 183);

</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/claw_left.png</normaloff>:/images/claw_left.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="clawRight">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>230</y>
          <width>91</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(183, 183, 183);
border-style: solid;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/claw_right.png</normaloff>:/images/claw_right.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="stopButton">
        <property name="geometry">
         <rect>
          <x>232</x>
          <y>110</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="settingsPage">
       <property name="styleSheet">
        <string notr="true">background: rgb(67, 83, 106)</string>
       </property>
       <widget class="QHoverSensitiveButton" name="backButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>81</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 0, 0, 0%);
border: solid;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/back.png</normaloff>:/images/back.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="hoverButton">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>160</y>
          <width>121</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>ON</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>170</y>
          <width>191</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Helvetica</family>
          <pointsize>29</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">qproperty-alignment: AlignCenter;
font: 29pt &quot;Helvetica&quot;;
color: white;</string>
        </property>
        <property name="text">
         <string>Hover Mode</string>
        </property>
       </widget>
       <widget class="QLabel" name="hover_time_label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>270</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Helvetica</family>
          <pointsize>24</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">qproperty-alignment: AlignCenter;
font: 24pt &quot;Helvetica&quot;;
color: white;</string>
        </property>
        <property name="text">
         <string>3.0 secs</string>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="hover_time_up_button">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>240</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(240, 223, 51);
font: 30pt &quot;.Helvetica Neue DeskInterface&quot;;
border: 1px solid black;</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="hover_time_down_button">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>290</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(29, 241, 240);
font: 30pt &quot;.Helvetica Neue DeskInterface&quot;;
border: 1px solid black;</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>240</y>
          <width>221</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Helvetica</family>
          <pointsize>29</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">qproperty-alignment: AlignCenter;
font: 29pt &quot;Helvetica&quot;;
color: white;</string>
        </property>
        <property name="text">
         <string>Hover Time</string>
        </property>
       </widget>
       <widget class="QHoverSensitiveButton" name="autoButton">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>90</y>
          <width>121</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>ON</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>90</y>
          <width>231</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Helvetica</family>
          <pointsize>29</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">qproperty-alignment: AlignCenter;
font: 29pt &quot;Helvetica&quot;;
color: white;</string>
        </property>
        <property name="text">
         <string>Auto Movement</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>0</y>
          <width>331</width>
          <height>91</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0,0,0,0%);
border: solid;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/settings-title.png</normaloff>:/images/settings-title.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tutorialPage">
       <widget class="QHoverSensitiveButton" name="backButton_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>91</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0,0,0,0%);
border: solid;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/back.png</normaloff>:/images/back.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>20</y>
          <width>491</width>
          <height>331</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background: rgba(0, 0, 0, 0%);
border-style: solid;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/tutorial_page.png</normaloff>:/images/tutorial_page.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>400</width>
          <height>400</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="countdown">
       <widget class="QLabel" name="countdownLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>601</width>
          <height>151</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>150</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background: rgb(67, 83, 106);
qproperty-alignment: AlignCenter;</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
       <widget class="QLabel" name="countdownLabel2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>180</y>
          <width>611</width>
          <height>151</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>50</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background: rgb(67, 83, 106);
qproperty-alignment: AlignCenter;</string>
        </property>
        <property name="text">
         <string>SECONDS REMAINING</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QHoverSensitiveButton</class>
   <extends>QPushButton</extends>
   <header>qhoversensitivebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
